src/api/spotify/service.ts

```diff
@@ -1,35 +1,46 @@
 import type { SongDetails, SearchDetails, ArtistDetails } from 'src/types/Details';
 import type { Song, Artist, LocalStorage } from 'src/api/spotify/types';
+import fetch from 'cross-fetch';
```

Comment:
please explain your research and reasoning behind why this is required - correct me if the linked webpage below is insufficient/incorrect

https://developer.mozilla.org/en-US/docs/Web/API/fetch

Reply:
It is needed to run tests because fetch is not native to the Node.js enviornent.